README.md

```diff
@@ -98,6 +110,27 @@ Create a new lion instance with the name `simba`
 
 ```ruby
 # code here
+class Animal
+  attr_accessor :name
+
+
+  def initialize name, greet
+    @name = name
+    @greet = greet
+  end
+
+
+end
+
+class Lion < Animal
+  @@pack = Animal
+
```

Comment:
rather than a method, king should be a property with a boolean (true/false) value